### PR TITLE
Update LockDealBundleProvider state

### LINKED ISSUE #106: un-nest
I think we can hold the `unit` directly

https://github.com/The-Poolz/LockDealNFT/blob/077d6e5656c4b67c24d9e793bea17dd1aa75c460/contracts/LockDealBundleProvider/LockDealBundleProviderState.sol#L9-L13

### PR DESCRIPTION
**Pull request by 0xSolcroot**
close #106

### CODE REVIEW

File: contracts/LockDealBundleProvider/LockDealBundleProviderState.sol [hunk 1]
```diff
@@ -6,9 +6,5 @@ import "../LockDealNFT/LockDealNFT.sol";
 /// @title LockDealBundleProviderState contract
 /// @notice Contains storage variables
 contract LockDealBundleProviderState {
-    mapping(uint256 => LockDealBundle) public poolIdToLockDealBundle;
-
-    struct LockDealBundle {
-        uint256 firstSubPoolId;
-    }
+    mapping(uint256 => uint256) public poolIdToLockDealBundle;
```
Comment (Lomet) [inline contracts/LockDealBundleProvider/LockDealBundleProviderState.sol:9]: I think its better to keep the `first` in the name:
maybe we can try:
`firstPoolIdToBundleId`

**Review by Lomet — COMMENTED** [review]
looks good. what about the name?

**Comment by 0xSolcroot** [comment]
> looks good. what about the name?

since `poolId` is a bundle pool Id, I think it's better not to use `first`

**Comment by Lomet** [comment]
> bundlePoolIdToFirstPoolId

yes. you right, 
`bundlePoolIdToFirstPoolId` sounds good

File: contracts/LockDealBundleProvider/LockDealBundleProviderState.sol (hunk 1 above)
Comment (0xSolcroot) [reply contracts/LockDealBundleProvider/LockDealBundleProviderState.sol]: Fixed with `bundlePoolIdToFirstPoolId`

**Review by Lomet — APPROVED** [review]
:)

**Pull request merged**